src/pages/registrering/grunnleggende-ferdigheter/grunnleggendeFerdigheter.less

```diff
@@ -0,0 +1,11 @@
+@import '../../../styles/variables.less';
+
+@media (min-width: @mobile-breakpoint) {
+  .grunnleggende-ferdigheter__alternativer {
```

Comment:
Jeg har brukt camelCase andre steder i appen for BEM-klassenavn, blir gjerne lettere å lese, spesielt om du har modifiers (`dette-er-en-knapp--stor` vs. `detteErEnKnapp--stor`)

```
.grunnleggendeFerdigheter__alternativer {

}
```

evt. med BEM-klassen øverst for å fremheve at vi scoper all CSS-en:

```
.grunnleggendeFerdigheter {

    @media (min-width: @mobile-breakpoint) {
        &__alternativer {
            // ...
        }
    }
}
```